gtsam/hybrid/tests/testDCGaussianMixtureFactor.cpp

```diff
@@ -88,6 +89,51 @@ TEST(DCGaussianMixtureFactor, Sum) {
   EXPECT(actual.at(1) == f22);
 }
 
+TEST(DCGaussianMixtureFactor, Printing) {
+  DiscreteKey m1(1, 2);
+  auto A1 = Matrix::Zero(2, 1);
+  auto A2 = Matrix::Zero(2, 2);
+  auto b = Matrix::Zero(2, 1);
+  auto f10 = boost::make_shared<JacobianFactor>(X(1), A1, X(2), A2, b);
+  auto f11 = boost::make_shared<JacobianFactor>(X(1), A1, X(2), A2, b);
+  std::vector<GaussianFactor::shared_ptr> factors{f10, f11};
+
+  DCGaussianMixtureFactor mixtureFactor({X(1), X(2)}, {m1}, factors);
+
+  std::string expected = \
+R"([ x1 x2; 1 ]{
+ Choice(1) 
+ 0 Leaf Jacobian factor on 2 keys: 
+  A[x1] = [
+	0;
+	0
+]
```

Comment:
this is not indented

Reply:
Do we need the indentation though? The output is pretty straightforward to parse and it has sufficient indentation.

But yeah, we can park this for later.

Reply:
Well, I'd merge it now, work has been done. Just don't dwell on it.